apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/V2Server.java

```diff
@@ -113,5 +113,7 @@ protected void initialize() throws ServletException {
     // Default to XML and pretty printing.
     setDefaultResponseEncoding(EncodingEnum.JSON);
     setDefaultPrettyPrint(false);
+
+    registerInterceptor(new TimerInterceptor());
```

Comment:
a comment here that this registers hapi interceptors to capture metrics on the timing characteristics of the response handling would be great (same comment on V1)

Reply:
Updating